### PR TITLE
PACKAGE UPDATE

### PR DESCRIPTION
**Pull request by bbwanjia**
# Description

- Please include a summary of the changes and the related issue.
- Please also include relevant motivation and context.
- List any dependencies that are required for this change.

Fixes # (issue)

## Type of change

- Please delete options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

# How Has This Been Tested?

- Please describe the tests that you ran to verify your changes.
- Provide instructions so we can reproduce. Please also list any relevant details for your test configuration

- [ ] Test A
- [ ] Test B

**Test Configuration**:
> Please copy `cpc -v` output here

```

```

# Checklist:

- [ ] My code follows the style of this project
- [ ] I have performed a self-review of my code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

**Comment by iewnfod** [comment]
Please state clearly what you have done in `package.py`

**Comment by iewnfod** [comment]
Also, I cannot see any test result to show that your code is proper

**Comment by lightumcc** [comment]
I recommend using `-pkg` as a command parameter instead of `-g`. This is easier for the user to remember

**Comment by bbwanjia** [comment]
> Please state clearly what you have done in `package.py`

The main function I passed to your options.py, is option_package(), which is too long and complicated, and I should put it in another file to make the logic clearer.

**Comment by bbwanjia** [comment]
> Also, I cannot see any test result to show that your code is proper

EXAMPLE COMMAND

List:
```
% cpc -g l
CPC official package manager
2 package(s) are enabled:
        exam[le
        egEND
0 package(s) are disabled:
```

Uninstall:

```
% cpc -g uninstall egEND
CPC official package manager
TIP: you can use `yes|<command>` to skip [y/n] choices
egEND is currently enabled.
Confirm to uninstall egEND? [y/n] n
Uninstallation cancelled.
```

Enable/disable
```
% cpc -g disable egEND
CPC official package manager
Disabled package egEND.
```

**Comment by bbwanjia** [comment]
> I recommend using `-pkg` as a command parameter instead of `-g`. This is easier for the user to remember

Rejected.
Abbreviation short hyphen command should always in ONE letter.
g is a stressed syllable in the word package, so easy to remenber.
consider, like pip, use cpg as shorthand. (Not yet implemented)

**Comment by iewnfod** [comment]
Can your disable actually working? You do not give any changes to the import statement. How could you do this?

File: packages/exam[le.cpg
Comment (iewnfod) [inline packages/exam[le.cpg:1]: Please rename this file into `example.cpg`

**Comment by lightumcc** [comment]
> > I recommend using `-pkg` as a command parameter instead of `-g`. This is easier for the user to remember
> 
> 
> 
> Rejected.
> 
> Abbreviation short hyphen command should always in ONE letter.
> 
> g is a stressed syllable in the word package, so easy to remenber.
> 
> consider, like pip, use cpg as shorthand. (Not yet implemented)
> 
> 

Yes but there are already some arguments that more than one character, we should first let user understand what they typed

**Comment by lightumcc** [comment]
Also I don't recommend you to put the disabled modules into a directory that starts with `.`. Hiding folders will make it more difficult for ordinary users to manage modules.

**Comment by iewnfod** [comment]
After some discussions, we think the function of package management should be done by another single tool rather than the core. You may create a project named as `cpm` which is stands for `cpc package manager`.

**Pull request closed without merging**